docs/sorting.md

```diff
@@ -96,6 +96,16 @@ With the combination of `ISearchResult.Skip` and `maxResults`, we can tell Lucen
 * Skip over a certain number of results without allocating them and tell Lucene
 * only allocate a certain number of results after skipping
 
+### Deep Paging
+When using Lucene.NET as the Examine provider it is possible to more efficiently perform deep paging.
+Steps:
+1. Build and execute your query as normal.
+2. Cast the ISearchResults from IQueryExecutor.Execute to ILuceneSearchResults
+3. Store ILuceneSearchResults.SearchAfter (SearchAfterOptions) for the next page. It may be worth serializing this class and cryptographically hashing it to prevent tampering in a web application so that it can be made available to the next request for the next page.
```

Comment:
I don't quite follow the comment about hashing here, can you describe the tampering situation?

Reply:
Removed the comment. Was thinking if the next page was encoded as a url parameter it might be worth preventing tampering. But I'll leave this to the implementor.